src/components/DeleteMovieForm/DeleteMovieForm.tsx

```diff
@@ -0,0 +1,31 @@
+import Dialog from "../Dialog/Dialog";
+import {useLoaderData, useNavigate} from "react-router-dom";
+import {DialogType} from "../../constants/DialogType";
+import axios from "axios";
+import DeleteDialog from "../DeleteDialog/DeleteDialog";
+
+async function deleteMovie(id: number) {
+    await axios.delete(`http://localhost:4000/movies/${id}`);
+}
+
+function DeleteMovieForm() {
+    const navigate = useNavigate();
+    const {movieDetails} = useLoaderData();
+
+    async function handleDeleteMovie(id: number) {
+        try {
+            await deleteMovie(id);
+            navigate(`/`, { replace: true });
```

Comment:
I deleted movie and I was redirect to main page. Would be better that my search that i did before deleating was keep 
![image](https://github.com/user-attachments/assets/92c49360-1fb5-4cb2-a703-4f85b8c8b432)

Reply:
the same for edit